JS_HW_1/JS_HW_1.3.js

```diff
@@ -0,0 +1,16 @@
+// Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), 
+// и выводит сумму равную n + nn + nnn, где n не перемножаются, а конкатенируются
+
+let n = '';
```

Comment:
What is the purpose of this variable ?

Reply:
Fixed the mistake: `n` is a function parameter, no need to declare it as a variable